### DISCUSSION TITLE
Include KC in the log list somehow

### DISCUSSION THREAD

**Discussion by Edpon** [evansloan/collection-log#54] [Ideas]
80% of the time i see people use !log, they then immediately follow it up with !kc. Very common to see people using !log to show how spooned or how dry they are, but context of their KC is needed. My clan was just having this discussion wishing it was a feature!

We hope you consider it
Thank you

**Comment by lkacoy** [comment]
This is not a bad shout. I like the idea as well. My clan also does this a lot.
